lib/auth_app_web/router.ex

```diff
@@ -54,32 +54,33 @@ defmodule AuthAppWeb.Router do
       on_mount: [{AuthAppWeb.UserAuth, :redirect_if_user_is_authenticated}] do
       live "/users/register", UserLive.Registration, :new
       live "/users/log-in", UserLive.Login, :new
-      live "/users/reset-password", UserLive.ForgotPassword, :new
-      live "/users/reset-password/:token", UserLive.ResetPassword, :edit
+      live "/users/log-in/:token", UserLive.Confirmation, :new
+      post "/users/log-in", UserSessionController, :create
     end
-
-    post "/users/log-in", UserSessionController, :create
   end
 
   scope "/", AuthAppWeb do
     pipe_through [:browser, :require_authenticated_user]
 
+    live_session :require_sudo_mode,
```

Comment:
Making it a different live_session means we require a full socket authentication and I don't think it is required in this case? I would declare the `on_mount` callback in this case in the Settings live itself.

Reply:
Although this may require moving confirm-email/:token to another Controller/LiveView.

Reply:
I need to find a way to redirect the user to the settings again after authentication. To do that with the current return_to mechanism, I'd need to go through a controller - so there's probably going to be more changes here...

Reply:
in dead land, this is really easy...
![image](https://github.com/user-attachments/assets/23c66846-6874-43e2-bb23-8a033dfab368)